client/state/reader/teams/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External Dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_TEAMS_REQUEST,
+	READER_TEAMS_REQUEST_FAILURE,
+	READER_TEAMS_REQUEST_SUCCESS,
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const items = createReducer( [], {
+	[ READER_TEAMS_REQUEST_SUCCESS ]: ( state, action ) => [ ...state, ...action.teams ],
```

Comment:
> it should be a full replacement.

really? maybe not. replacement wipes history while merging preserves it.

another question I'd ask is will we need to store any information about these teams or only know how the slug and title relate?